js/valid.js

```diff
@@ -0,0 +1,21 @@
+const form = document.getElementById('form');
+const email = document.getElementById('mail');
+const alertParent = document.querySelector('.btn-submit-alert');
+const small = document.createElement('small');
+alertParent.appendChild(small);
+small.innerText = 'Please enter a valid email address.';
+small.style.color = '#fff';
+small.style.backgroundColor = '#e74c3c';
+small.style.padding = '12px';
+small.style.visibility = 'hidden';
```

Comment:
- [ ] As per milestone requirements, The error message fits the design from the chosen template in Figma (e.g., font, colours, images, tests, margins). Kindly try to follow the design pattern and change the font colour and background colour as they fit to design.

![image](https://user-images.githubusercontent.com/53019916/131842194-af03db97-89a4-47a2-9783-237a10c27409.png)

Reply:
Hello hemant-soni-vst-au4 
 May you please check my changes and tell me if everything is okay?